server/portal/apps/tickets/unit_test.py

```diff
@@ -4,7 +4,24 @@ def test_tickets_get(client, authenticated_user):
     assert response.url == '/workbench/dashboard/'
 
 
-def test_ticket_create_authenticated(client, authenticated_user):
+def test_ticket_create_authenticated(client, regular_user):
```

Comment:
why not authenticated_user instead of regular_user fixture here? and remove `force_login`line